apix-files/README.md

```diff
@@ -0,0 +1,40 @@
+# API Explorer file server
+
+API Explorer (aka APIX) can be uses some JSON-formatted "index" files to augment the information provided in a specification.
```

Comment:
> API Explorer (aka APIX) can be uses some 

?

> API Explorer (aka APIX) uses some 

instead?

Reply:
Well, it's definitely wrong now. I'll fix.